idurar/idurar-erp-crm
# Fix: missing translation keys

## Description

I was working on this PR #975 when I noticed that the search key is missing for the locale files in all the languages. So I added it in all the languages.

## Related Issues

No related issues

## Steps to Test

1. Navigate to a page where the search (Company page for example)
2. Change language and check if the search input field changed it's text
**Important:** Make sure to delete anything cached in Local Storage before testing

## Screenshots (if applicable)

Before PR:
![image](https://github.com/idurar/idurar-erp-crm/assets/12747343/c5f61fcc-3ac3-4200-a793-d880d8f1b969)

After PR:
![image](https://github.com/idurar/idurar-erp-crm/assets/12747343/bcf49099-5bee-4954-ab38-4bd08d48fcc0)


## Checklist
